docs/config/Containerfile.tinyproxy

```diff
@@ -1,13 +1,16 @@
-FROM docker.io/alpine:latest
+FROM registry.access.redhat.com/ubi9/ubi-minimal
 
-RUN apk add --no-cache tinyproxy \
- && apk add --no-cache nano
+# The tinyproxy package is only available in EPEL7
```

Comment:
Is there some more modern replacement?

Reply:
There is, but not in "official" repositories. At least, not something I could find.
The alternative is to use more complex proxies like nginx, etc., or build tinyproxy from sources.

Reply:
I don't think that's really necessary. I don't think we would add EPEL instructions to product docs anyway, but it's OK in the project docs.